core/modules/book/tests/book.test

```diff
@@ -95,12 +95,38 @@ class BookTestCase extends BackdropWebTestCase {
 
     // Check that book pages display along with the correct outlines and
     // previous/next links.
-    $this->checkBookNode($book, array($nodes[0], $nodes[3], $nodes[4]), FALSE, FALSE, $nodes[0], array());
-    $this->checkBookNode($nodes[0], array($nodes[1], $nodes[2]), $book, $book, $nodes[1], array($book));
-    $this->checkBookNode($nodes[1], NULL, $nodes[0], $nodes[0], $nodes[2], array($book, $nodes[0]));
-    $this->checkBookNode($nodes[2], NULL, $nodes[1], $nodes[0], $nodes[3], array($book, $nodes[0]));
-    $this->checkBookNode($nodes[3], NULL, $nodes[2], $book, $nodes[4], array($book));
-    $this->checkBookNode($nodes[4], NULL, $nodes[3], $book, FALSE, array($book));
+    $this->checkBookNode($book, array(
+      $nodes[0],
+      $nodes[3],
+      $nodes[4],
+    ), FALSE, FALSE, $nodes[0], array(
+      $book,
+    ));
+    $this->checkBookNode($nodes[0], array(
+      $nodes[1],
+      $nodes[2],
+    ), $book, $book, $nodes[1], array(
+      $book,
+      $nodes[0],
+    ));
+    $this->checkBookNode($nodes[1], NULL, $nodes[0], $nodes[0], $nodes[2], array(
+      $book,
+      $nodes[0],
+      $nodes[1],
+    ));
+    $this->checkBookNode($nodes[2], NULL, $nodes[1], $nodes[0], $nodes[3], array(
+      $book,
+      $nodes[0],
+      $nodes[2],
+    ));
+    $this->checkBookNode($nodes[3], NULL, $nodes[2], $book, $nodes[4], array(
+      $book,
+      $nodes[3],
+    ));
+    $this->checkBookNode($nodes[4], NULL, $nodes[3], $book, FALSE, array(
+      $book,
+      $nodes[4],
+    ));
```

Comment:
I'm assuming that PHPCS was complaining about the formatting here (things like "the array exceeds 80 characters" etc.), however the way it is now is much harder to read and get an overview of what's happening than before.

Can we please do either one of these two things:
- Suppress the PHPCS nagging using a set of `// phpcs:disable`/`// phpcs:disable` before/after these lines and otherwise leave the formatting/restructuring unchanged.
- Use some helper variables that make it obvious what the parameters are each time `$this->checkBookNode()` is called.